addons/account/models/account_tax.py

```diff
@@ -524,7 +524,7 @@ def compute_all(self, price_unit, currency=None, quantity=1.0, product=None, par
         if not self:
             company = self.env.company
         else:
-            company = self[0].company_id._accessible_branches()[:1]
+            company = self[0].company_id._accessible_branches()[:1] or self[0].company_id
```

Comment:
To me `_accessible_branches` should always return a result... :thinking:

Reply:
If it doesn't return something, it means that you would have an access error when trying to read `self[0].company_id` anyways...